.github/workflows/unit_tests.yaml

```diff
@@ -61,5 +61,10 @@ jobs:
           kubectl create secret generic postgres-secrets --from-literal=password='super-secure-password'
           kubectl create secret generic seqr-secrets --from-literal=django_key='securely-generated-key'
 
-          # The extra variable is to allow kubernetes to ping our postrgres running locally
+          # The extra variable is to allow kubernetes to ping our postgres running locally
+          # From a container, the IP address of the host machine is 172.17.0.1
           ct install --charts charts/seqr --namespace default --helm-extra-set-args "--set=environment.POSTGRES_SERVICE_HOSTNAME=172.17.0.1"
+      - name: Test Hail Search Installation
+        run: |
```

Comment:
This is going to fail for the time being because of an issue (ish) with the hail search code.